solutions/src/main/java/ru/sberbank/school/task02/ExtendedFxConversionServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+public class ExtendedFxConversionServiceImpl extends FxConversionServiceImpl implements ExtendedFxConversionService {
+
+    public ExtendedFxConversionServiceImpl(ExternalQuotesService externalQuotesService) {
+        super(externalQuotesService);
+    }
+
+    @Override
+    public Optional<BigDecimal> convertReversed(ClientOperation operation, Symbol symbol,
+                                                BigDecimal amount, Beneficiary beneficiary) {
+        return convertReversed(operation, symbol, amount, 0, beneficiary);
+    }
+
+    @Override
+    public Optional<BigDecimal> convertReversed(ClientOperation operation, Symbol symbol, BigDecimal amount, double delta, Beneficiary beneficiary) {
```

Comment:
Line is longer than 120 characters (found 149).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a860579825>*